benchmark/text-generation-inference/llama3-70b-inf2.48xlarge/docker-compose.yaml

```diff
@@ -0,0 +1,31 @@
+version: '3.7'
+
+services:
+  tgi-1:
+    image: neuronx-tgi:latest
+    ports:
+      - "8080:8080"
+    environment:
+      - PORT=8080
+      - MODEL_ID=${MODEL_ID}
+      - HF_BATCH_SIZE=${HF_BATCH_SIZE}
+      - HF_SEQUENCE_LENGTH=${HF_SEQUENCE_LENGTH}
```

Comment:
```suggestion
```

Reply:
Let me test this and I can clean it up everywhere.